### PR TITLE
[B5] Exports: create new export

### PR DESCRIPTION
**Pull request by orangejenny**
## Product Description
Joint effort with @AmitPhulera 

User-facing changes are pretty minor.

### Before

Screenshot 2024-10-03 at 7 39 15 AM
Screenshot 2024-10-03 at 7 39 45 AM
Screenshot 2024-10-03 at 7 39 55 AM

### After

Screenshot 2024-10-03 at 7 26 25 AM
Screenshot 2024-10-03 at 7 26 41 AM
Screenshot 2024-10-03 at 7 27 17 AM

### Safety story
UI-level changes. Requesting QA.

### Automated test coverage

I don't think so

### QA Plan

https://dimagi.atlassian.net/browse/QA-7132 - joint QA with https://github.com/dimagi/commcare-hq/pull/35161

### Rollback instructions
- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Comment by orangejenny** [comment]
Tests fail because I'm holding off on running `build_bootstrap5_diffs` due to potential conflicts with https://github.com/dimagi/commcare-hq/pull/35161

I'm not cleaning up the old B3 templates & updating migration status for the same reason, I'll do that in a later PR.

**Review by millerdev — COMMENTED** [review]
Commits up to https://github.com/dimagi/commcare-hq/pull/35170/commits/aefedd401dff6526dfab72b318b8c7de664c1586 look good.

I assume you're planning to add an additional commit after running `build_bootstrap5_diffs` just before this is ready to merge. Is that correct?

**Comment by orangejenny** [comment]
@millerdev yes

**Comment by orangejenny** [comment]
QA passed and https://github.com/dimagi/commcare-hq/pull/35161 is merged. I added the B5 diffs, this is now ready to go.

File: corehq/apps/export/static/export/js/utils.js
Comment (gherceg) [inline corehq/apps/export/static/export/js/utils.js:13]:
```suggestion
        var cls = window.USE_BOOTSTRAP5 ? 'badge' : 'label';
```

File: corehq/apps/export/static/export/js/utils.js
Comment (gherceg) [inline corehq/apps/export/static/export/js/utils.js:15]:
```suggestion
            return cls + (window.USE_BOOTSTRAP5 ? ' text-bg-warning' : ' label-warning');
```

**Comment by orangejenny** [comment]
@gherceg Thank you. Fixed tests for real in d3e0ea131b9

**Review by gherceg — APPROVED** [review]

**Pull request merged**
